io/prometheus/client/metrics.proto

```diff
@@ -107,6 +107,8 @@ message Histogram {
   // histograms.
   repeated sint64 positive_delta    = 13; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
   repeated double positive_count    = 14; // Absolute count of each bucket.
+
+  repeated Exemplar exemplars = 16;
```

Comment:
This needs a comment along the lines: "Only used for native histograms. These exemplars MUST have a timestamp."

Reply:
Done